style.css

```diff
@@ -0,0 +1,83 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+
+  gap: 5rem;
+}
+
+body {
+  font-family: sans-serif;
+  color: grey;
+  font-weight: 300;
+  line-height: 1.6;
+}
+
+.fa-heart {
+  color: red;
+}
+.fa-canadian-maple-leaf {
+  color: black;
+}
+.fa-gem {
+  color: red;
+}
+.fa-candy-cane {
+  color: black;
+}
```

Comment:
You can combine similar classes that share same properties like with the icons there are 2 reds and 2 blacks which can be merged together:
```css
.fa-heart,
.fa-gem {
  color: red;
}
.fa-canadian-maple-leaf,
.fa-candy-cane {
  color: black;
}
```